pkg/reconciler/common/image_override.go

```diff
@@ -0,0 +1,71 @@
+package common
+
+import (
+	"os"
+	"strings"
+
+	"knative.dev/operator/pkg/apis/operator/v1alpha1"
+)
+
+const imagePrefix = "IMAGE_"
```

Comment:
Maybe the prefix is a bit to generic and there is a certain likelihood that there might be an accidental environment variable set that also start with `IMAGE_` ? Maybe sth like "SLO_IMAGE" might be less dangerous ? (SLO for "ServerLess-Operator")

Reply:
Hmm, I'd rather not change this given it's the same throughout the serverless-operator too. I think we have enough "control" over the upstream to at least know if and when this prefix would collide.

Reply:
ok